app/javascript/packs/pages/BrowseRequests.jsx

```diff
@@ -22,7 +22,10 @@ const useStyles = makeStyles(theme => ({
 
 const TOTAL_PROVIDERS = gql`
   {
-    providers(filters: { active: true }, orderBy: { direction: ASC, sort: CITY }) {
+    providers(filters: {
+      active: true
+      updatedWithinDays: 15
```

Comment:
Learning graphql. Is there a way to pass an operator here so that `filter_helpers` does not need to be updated?

Was hoping for something like `updated_at > ? DATE_HERE`.

Reply:
I don't think so.